jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -179,6 +182,21 @@ async def auth_login(self):
         response = await self.client.fetch(
             f"{self.address}/auth/_login", method="PUT", body="{}"
         )
+
+        resp_json = json.loads(response.body.decode())
+        # may bubble exception up to handler if oidc_state not in response
+        oidc = resp_json['oidc_state']
+
+        asyncio.create_task(self.auth_login_token(oidc))
```

Comment:
Does this need to be awaited?

Reply:
no, we specifically do not want to await this since we want to redirect the user to the login page without waiting for their login to complete. i believe `auth_login_token` will only finish once the user completes the login. this is similar to what is done before within the mount server by creating the authenticate go routine and updating the token without waiting on it.

Reply:
perhaps i should capture this in a comment